tests/test_connections/test_ledger/test_ledger.py

```diff
@@ -471,7 +474,6 @@ async def test_not_hanging_with_multiplexer(self) -> None:
 
         # `receive()` should be done,
         # and multiplexer's `_receiving_loop` should have put the `normal_dummy_envelope` in the `in_queue`
-        self.multiplexer.disconnect()
```

Comment:
Feel free to add this, if necessary.

Reply:
It should not be necessary, it was just a residue from a previous commit that was demonstrating an issue.

Reply:
Yeah let's also remove the comment then :)